test/data/legalDeclationHTML.html

```diff
@@ -37,117 +37,127 @@
+                        <h3 class="govuk-heading-s declaration-subheader">Executors applying for probate</h3>
+                        <p class="govuk-body declaration-item">I am an executor named in the will as Jason Smith, and I am applying for probate.</p>
+                        <p class="govuk-body declaration-item">I will send to the probate registry what I believe to be the true and original last will and testament of Mike  Samuels.</p>
+                        <h2 class="govuk-heading-l declaration-subheader">Declaration</h2>
+                        <p class="govuk-body declaration-item">I confirm that I will administer the estate of Mike  Samuels, according to law. I will:</p>
+                        <ul class="govuk-list govuk-list--bullet">
                             <li>collect the whole estate</li>
                             <li>keep full details (an inventory) of the estate</li>
                             <li>keep a full account of how the estate has been administered</li>
                         </ul>
-                        <p class="declaration-item">If the probate registry (court) asks me to do so, I will:</p>
-                        <ul class="list list-bullet">
+                        <p class="govuk-body declaration-item">If the probate registry (court) asks me to do so, I will:</p>
+                        <ul class="govuk-list govuk-list--bullet">
                             <li>provide the full details of the estate and how it has been administered</li>
                             <li>return the grant of probate to the court</li>
                         </ul>
-                        <p class="declaration-item">I understand that:</p>
-                        <ul class="list list-bullet">
+                        <p class="govuk-body declaration-item">I understand that:</p>
+                        <ul class="govuk-list govuk-list--bullet">
                             <li>my application will be rejected if I do not answer any questions about the information I have given</li>
                             <li>criminal proceedings for fraud may be brought against me if I am found to have been deliberately untruthful or dishonest</li>
                         </ul>
                     </div>
                     <div class="declaration-grid-item">
-                        <h2 class="heading-medium">Save this page</h2>
-                        <p>
+                        <h2 class="govuk-heading-m">Save this page</h2>
+                        <p class="govuk-body">
                             <i class="icon icon-file-download"></i>&nbsp;
                             <a id="declarationPdfHref" href="/declaration-pdf">Legal declaration (8kb PDF)</a>
```

Comment:
missing class

Reply:
Good spot but irrelevant, this file is only used for a test... :)